src/app/thrift-services/damsel/domain-cache.service.ts

```diff
@@ -61,17 +60,26 @@ export class DomainCacheService {
             )
         );
 
-    commit = (commit: Commit, version?: Version) => {
-        return (version ? of(version) : this.version$).pipe(
-            take(1),
+    commit = (commit: Commit, version?: Version | number, reload = true) => {
+        const version$ = version ? of(version) : this.version$.pipe(take(1));
+        return version$.pipe(
             switchMap((v) =>
```

Comment:
can we name variables not in one letter?

Reply:
here is a short context, we are practicing this code